disttask/importinto/dispatcher.go

```diff
@@ -590,12 +594,22 @@ func (h *flowHandle) failJob(ctx context.Context, handle dispatcher.TaskHandle,
 	taskMeta *TaskMeta, logger *zap.Logger, errorMsg string) error {
 	h.switchTiKV2NormalMode(ctx, logger)
 	h.unregisterTask(ctx, gTask)
+	redactSensitiveInfo(gTask, taskMeta)
 	return handle.WithNewSession(func(se sessionctx.Context) error {
 		exec := se.(sqlexec.SQLExecutor)
 		return importer.FailJob(ctx, exec, taskMeta.JobID, errorMsg)
 	})
 }
 
+func redactSensitiveInfo(gTask *proto.Task, taskMeta *TaskMeta) {
+	taskMeta.Stmt = ""
+	taskMeta.Plan.Path = ast.RedactURL(taskMeta.Plan.Path)
```

Comment:
After looking at the redactURL implementation, which currently only supports s3, do we need to consider the gcs case?

https://github.com/pingcap/tidb/blob/8633c8ce9921bf17a92b582a6b5d299d95213ac6/parser/ast/misc.go#L3438

Reply:
no

Reply:
it's for url like `s3://`, not http